harness/determined/common/experimental/user.py

```diff
@@ -118,3 +119,56 @@ def _from_bindings(cls, user_bindings: bindings.v1User, session: api.Session) ->
         user = cls(session=session, user_id=user_bindings.id)
         user._hydrate(user_bindings)
         return user
+
+
+class TokenType(enum.Enum):
```

Comment:
If you do go with the `det token *` cli pattern instead of `det user token *`, I'd say this should probably be a file named `token.py`, not part of `user.py`.  Especially since you can only do `Determined.list_tokens()` (implying that you can only list your own tokens) and not `User.list_tokens()` (which would imply that like, an admin could list other users' tokens).

Actually... is it intended to allow admins to list tokens for their users?

Reply:
Yes, only `ClusterAdmins` has permission to list the token info of other users

Reply:
How would they do that with this SDK design?  Or with this CLI design?

Reply:
CLI design would look like this for an Admin:

```
% det user token ls --all                                                                   

   ID |   User ID | Description      | Created At                  | Expires At                  | Revoked   | Token Type
------+-----------+------------------+-----------------------------+-----------------------------+-----------+------------------
 5656 |      2305 |                  | 2024-09-18T05:12:48.424089Z | 2024-09-18T05:12:50.424089Z | True      | ACCESS_TOKEN
 5662 |         1 |                  | 2024-09-18T05:41:55.342039Z | 2024-09-18T05:41:57.342039Z | True      | ACCESS_TOKEN
 5673 |      2305 |                  | 2024-09-18T17:50:02.784704Z | 2024-09-18T17:50:04.784704Z | False     | ACCESS_TOKEN
 5675 |         1 |                  | 2024-09-18T17:59:52.688643Z | 2024-09-18T17:59:54.688643Z | False     | ACCESS_TOKEN
...
```

Reply:
Resolving it as it is now implemented 
1. Moved from `det user token *` to `det token *` cli pattern 
2. We do not include SDK designs anymore in the PR